implants/imix/src/install.rs

```diff
@@ -31,17 +32,31 @@ pub async fn install() {
             // Run tome
             #[cfg(debug_assertions)]
             log::info!("running tome {embedded_file_path}");
-            let mut runtime = eldritch::start(Tome {
-                eldritch,
-                parameters: HashMap::new(),
-                file_names: Vec::new(),
-            })
+            let mut runtime = eldritch::start(
+                0,
+                Tome {
+                    eldritch,
+                    parameters: HashMap::new(),
+                    file_names: Vec::new(),
+                },
+            )
             .await;
             runtime.finish().await;
 
-            let _output = runtime.collect_text().join("");
             #[cfg(debug_assertions)]
-            log::info!("{_output}");
+            let mut output = String::new();
+
+            #[cfg(debug_assertions)]
+            for msg in runtime.collect() {
+                if let Message::ReportText(m) = msg {
+                    if let Err(err) = output.write_str(m.text().as_str()) {
+                        #[cfg(debug_assertions)]
+                        log::error!("failed to write text: {}", err);
+                    }
+                }
+            }
+            #[cfg(debug_assertions)]
+            log::info!("{output}");
```

Comment:
It's unclear what's happening here - but I think we want imix install to print in realtime similar to golem.

Reply:
This does not change any behavior, but currently this will print all output after evaluation. The `print_stdout` is a feature flag we likely don't want enabled for imix.